### PR TITLE
Feature/drone physics

### PR DESCRIPTION
**Pull request by Segelzwerg**

### CODE REVIEW

**Comment by Segelzwerg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/drone_controller/input_layer/test_drone_metrics.py  1
- src/drone_controller/input_layer/drone_metrics.py  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/treeandsea/DroneController/pullRequest?prid=5306106&bid=17365803)

**Comment by Segelzwerg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/drone_controller/input_layer/test_drone_metrics.py  1
- src/drone_controller/input_layer/drone_metrics.py  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/treeandsea/DroneController/pullRequest?prid=5306106&bid=17365803)

**Comment by Segelzwerg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/drone_controller/input_layer/test_drone_metrics.py  2
- src/drone_controller/input_layer/drone_metrics.py  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/treeandsea/DroneController/pullRequest?prid=5306106&bid=17365803)

File: src/drone_controller/input_layer/drone_metrics.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import math
+
+
+class DroneMetrics:
```
Comment (TreeKid) [inline src/drone_controller/input_layer/drone_metrics.py:4]: Maybe Physics or Measurements would be more clear

**Pull request merged**